docs/pages/connect-your-client/introduction.mdx

```diff
@@ -250,6 +240,38 @@ cluster](../../img/use-teleport/kubernetes-login.png)
 </TabItem>
 </Tabs>
 
+### Transferring files
+
+<Tabs>
+<TabItem label="tsh scp">
+
+`tsh scp` will let you transfer files to servers behind Teleport:
+
+```code
+$ tsh scp some-file.ext server.example.com:
+some-file.ext   7% |███████                | (25/342 MB, 2.9 MB/s) [9s:1m48s]
+```
+</TabItem>
+<TabItem label="Teleport Connect">
+
+Teleport Connect allows you to transfer files to a remote server and runs on MacOS, Linux and Windows:
+
+![Download with connect](../../img/download-with-connect.png)
+
+You can upload and download files from systems using drag and drop:
+
+![Drag and drop in WebUI](../../img/teleport-connect-drag-drop.png)
+
+</TabItem>
+<TabItem label="Web UI">
+
+You can also use the arrows in the top-left corner of the Web UI to upload and download files:
```

Comment:
```suggestion
You can use the arrows in the top-left corner of the Web UI to upload and download files:
```

It'd be worth to mention that in the tab for Connect as well, since the UI for accessing file transfers is pretty much the same.